MetaMask/metamask-extension
# ConfirmImportToken: Convert to Functional Component; Add storybook controls

Fixes: n/a

Explanation:  Similarly to https://github.com/MetaMask/metamask-extension/pull/13377 and https://github.com/MetaMask/metamask-extension/pull/13526, we are converting this component to a Functional Component.

Additionally:
- add jest tests
- add storybook controls

Manual testing steps:  
  - On the main page and underneath the list of accounts, click "import tokens"
  - Search any token and select it
  - Click "Next" 
  - Test this "Import Tokens" page works as expected
